src/main/scala/org/broadinstitute/dsde/firecloud/model/MethodEntity.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.firecloud.model
+
+import com.wordnik.swagger.annotations.{ApiModel, ApiModelProperty}
+import spray.json.DefaultJsonProtocol
+
+import scala.annotation.meta.field
+
+object MethodEntityJsonProtocol extends DefaultJsonProtocol {
+  implicit val impMethodEntity = jsonFormat10(MethodEntity)
+}
+
+@ApiModel(value = "Agora Method")
```

Comment:
Before we get too far in our own model classes ... can we share models with Agora via a common lib?